docs/notebooks/.ipynb_checkpoints/WMD_tutorial-checkpoint.ipynb

```diff
@@ -0,0 +1,601 @@
+     "text": [
+      "Cell took 158.570173 seconds to run.\n"
+     ]
+    }
+   ],
+   "source": [
+    "start = time()\n",
+    "\n",
+    "from gensim.similarities import WmdSimilarity\n",
+    "instance = WmdSimilarity(wmd_corpus, model, num_best=10)\n",
+    "\n",
+    "print 'Cell took %f seconds to run.' %(time() - start)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `num_best` parameter decides how many results the queries return. Now let's try making a query. The output is a list of indeces and similarities of documents in the corpus, sorted by similarity. Note that the output format is slightly different when `num_best` is `None` (i.e. not assigned).\n",
+    "\n",
+    "The query below is taken directly from one of the reviews in the corpus. Let's see if there are other reviews that are similar to this one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "sent = 'Very good, you should seat outdoor.'\n",
+    "\n",
+    "# Pre-process query.\n",
+    "query = sent.lower()  # Lower the text.\n",
+    "query = word_tokenize(query)  # Split into words.\n",
+    "query = [w for w in query if not w in stop_words]  # Remove stopwords.\n",
+    "query = [w for w in query if w.isalpha()]  # Remove numbers and punctuation.\n",
+    "\n",
+    "sims = instance[query]  # A query is simply a \"look-up\" in the similarity class."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The query and the most similar documents, together with the similarities, are printed below. We see that the retrieved documents are discussing the same thing as the query, although using different words. The query talks about getting a seat \"outdoor\", while the results talk about sitting \"outside\", and one of them says the restaurant has a \"nice view\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Query:\n",
+      "Very good, you should seat outdoor.\n",
+      "\n",
+      "sim = -0.7492\n",
+      "It's a great place if you can sit outside in good weather.\n",
+      "\n",
+      "sim = -0.7537\n",
+      "It was good I like the outside\n",
+      "\n",
+      "sim = -0.8371\n",
+      "nice view, good service\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print the query and the retrieved documents, together with their similarities.\n",
+    "print 'Query:'\n",
+    "print sent\n",
+    "for i in range(3):\n",
+    "    print\n",
+    "    print 'sim = %.4f' % sims[i][1]\n",
+    "    print documents[sims[i][0]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's try a different query, also taken directly from one of the reviews in the corpus."
```

Comment:
which corpus? the wmd corpus or w2v corpus? Do you mean "taken from one of the reviews of this restuarant"? If yes, then why don't we get the exact review back as the first match?